server/routes/activitiesMigration/activitiesMigrationController.ts

```diff
@@ -218,9 +218,7 @@ export default class ActivitiesMigrationController {
         nomisPayRatesUnknownIncentive,
         nomisActivitiesWithoutScheduleRules,
       ]) => {
-        req.session.startActivitiesMigrationForm.estimatedCount = (
-          estimatedCount - nomisActivitiesWithoutScheduleRules.length
-        ).toLocaleString()
```

Comment:
I thought it would be nice to exclude activities we know cannot be migrated from the estimate. However I've now realised the estimate is recalculated on the server side in some generic code I can't change without major hacking. Which I don't want to do.

Instead I've decided to include activities with missing schedules in the estimate but make it clear why they were ignored in the app insights query that show the migration results (see the [API change](https://github.com/ministryofjustice/hmpps-prisoner-from-nomis-migration/pull/1457)).

Reply:
Should be fine as this is a dev or internal users UI anyway